app/dv/pom.xml

```diff
@@ -236,8 +233,8 @@
     <!-- Common dependencies -->
     <version.springfox>2.9.2</version.springfox>
 
-    <version.org.mockito>2.21.0</version.org.mockito>
-    <version.com.fasterxml.jackson>2.9.9</version.com.fasterxml.jackson>
+    <version.org.mockito>${mockito.version}</version.org.mockito>
+    <version.com.fasterxml.jackson>${jackson.version}</version.com.fasterxml.jackson>
```

Comment:
I kinda like the `version.groupId` style of property naming better than what we use currently in the Syndesis parent POM.

Reply:
I agree.   As I make any further changes, I'll try to transition to this format.